geopandas/io/tests/test_file.py

```diff
@@ -363,6 +367,7 @@ def test_to_file_types(tmpdir, df_points, engine):
     df.to_file(tempfilename, engine=engine)
 
 
+@pytest.mark.skipif(not PYOGRIO_GE_06, reason="pyogrio < 0.6.0 doesn't support Int32")
```

Comment:
Here and below, should this skip be moved into the test body, depending on value of `engine`, so that Fiona is still tested?